resources/image_test.go

```diff
@@ -536,6 +578,14 @@ func TestImageOperationsGolden(t *testing.T) {
 	c.Assert(err, qt.IsNil)
 	dirinfos2, err := d2.Readdir(-1)
 
+	// sort entries of both dirs for stable tests, don't rely on the fs to handle sorting
+	sort.Slice(dirinfos1, func(i, j int) bool {
+		return strings.Compare(dirinfos1[i].Name(), dirinfos2[i].Name()) < 2
+	})
+	sort.Slice(dirinfos2, func(i, j int) bool {
+		return strings.Compare(dirinfos2[i].Name(), dirinfos2[i].Name()) < 2
```

Comment:
Do we need these?

I was under the impression that `Readdir` had a defined order (it says so in the docs) and that it was stable across OSes. I have not seen any test failures with this?

Also, the less funcs used does not look right.

Reply:
This wasn't needed for this PR. I removed it and force-pushed the changes.